pyfixest/demean.py

```diff
@@ -62,7 +62,9 @@ def demean_model(
                 if var_diff.ndim == 1:
                     var_diff = var_diff.reshape(len(var_diff), 1)
 
-                YX_demean_new = algorithm.residualize(var_diff)
+                weights = np.ones(YX.shape[0])
```

Comment:
Weighted least squares is not yet implemented. With the current implementation of the algo, weights need to be provided -> vector of ones.